components/CommunityList/CommunityListCard.js

```diff
@@ -1,13 +1,5 @@
-import isString from 'lodash/isString';
-
-import { DefaultCard } from 'ui-kit';
+import { HorizontalHighlightCard } from 'ui-kit';
 
 export default function CommunityListCard(props = {}) {
-  // Note: Safe assumption that `props.src` is an image from our own
-  // API and Cloudinary setup... so append parameters to optimize loading
-  const coverImage = isString(props.coverImage)
-    ? `${props.coverImage}&w=768`
-    : props.coverImage;
-
-  return <DefaultCard {...props} coverImage={coverImage} />;
+  return <HorizontalHighlightCard as="a" type="HIGHLIGHT_SMALL" {...props} />;
```

Comment:
Hey @drewbarontini and @dzwood , I'm not sure how I feel about this `<CommunityListCard>` "alias" component.
On the one hand, it's a simple little abstraction between the `<CommunityList>` and what card ultimately gets used to render it.

On the other, feels unnecessary 😕.

Thoughts?

Reply:
Hm. I'm fine with it as an alias, but could be convinced otherwise 🤷‍♂️

Reply:
^^^ yeah I'm pretty much good with either one. Probably leaning more towards it being unnecessary, but I'm fine with it.